cscs-checks/libraries/gridtools/gridtools_base.py

```diff
@@ -0,0 +1,164 @@
+                    },
+                    'daint:gpu': {
+                        'wall_time': (3800, None, 0.1, 'ms')
+                    },
+                    '*': {
+                        'wall_time': (0, None, None, 'ms')
+                    }
+                }
+            },
+            'vertical_advection_dycore_mc': {
+                'executable_opts': ['150', '150', '150'],
+                'reference': {
+                    'daint:mc': {
+                        'wall_time': (3500, None, 0.1, 'ms')
+                    },
+                    'daint:gpu': {
+                        'wall_time': (3700, None, 0.1, 'ms')
+                    },
+                    '*': {
+                        'wall_time': (0, None, None, 'ms')
+                    }
+                }
+            },
+            'simple_hori_diff_naive': {
+                'executable_opts': ['100', '100', '100'],
+                'reference': {
+                    'daint:mc': {
+                        'wall_time': (3200, None, 0.1, 'ms')
+                    },
+                    'daint:gpu': {
+                        'wall_time': (3700, None, 0.1, 'ms')
+                    },
+                    '*': {
+                        'wall_time': (0, None, None, 'ms')
+                    }
+                }
+            },
+            'simple_hori_diff_mc': {
+                'executable_opts': ['100', '100', '100'],
+                'reference': {
+                    'daint:mc': {
+                        'wall_time': (3300, None, 0.1, 'ms')
+                    },
+                    'daint:gpu': {
+                        'wall_time': (3700, None, 0.1, 'ms')
+                    },
+                    '*': {
+                        'wall_time': (0, None, None, 'ms')
+                    }
+                }
+            },
+            'vertical_advection_dycore_cuda': {
+                'executable_opts': ['200', '200', '200'],
+                'reference': {
+                    'daint:gpu': {
+                        'wall_time': (12000, None, 0.1, 'ms')
+                    },
+                    '*': {
+                        'wall_time': (0, None, None, 'ms')
+                    }
+                }
+            },
+            'simple_hori_diff_cuda': {
+                'executable_opts': ['150', '150', '150'],
+                'reference': {
+                    'daint:gpu': {
+                        'wall_time': (19000, None, 0.1, 'ms')
+                    },
+                    '*': {
+                        'wall_time': (0, None, None, 'ms')
+                    }
+                }
+            }
+        }
+
+        self.build_system.make_opts = [variant]
+        self.executable = os.path.join('regression', variant)
+        self.executable_opts = self.variant_data[variant]['executable_opts']
+        self.reference = self.variant_data[variant]['reference']
+
+        self.tags = {'production'}
```

Comment:
I don't think we should run these tests daily in production, since the purpose is not to be a CI for GridTools. The tags should be just `scs` and `benchmarking`.